Cargo.toml

```diff
@@ -84,13 +84,9 @@ use_self = "allow"
 
 [dependencies]
 # Starknet dependencies
-cainome = { git = "https://github.com/cartridge-gg/cainome.git", tag = "v0.2.5", default-features = false, features = [
-  "abigen-rs",
-] }
+cainome = { git = "https://github.com/cartridge-gg/cainome.git", tag = "v0.2.5", default-features = false, features = ["abig-rs"] }
```

Comment:
Why the `features` name is modified here?

Reply:
yeah this was a typo. Sorry I didn't realise it.

Reply:
Should I run format on the `Cargo.toml` as well for consistency, or make it like it was earlier i.e. some features on new line in extended form, while few others on the same line.